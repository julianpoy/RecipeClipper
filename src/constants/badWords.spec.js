import {
  badWords,
  generalBadWords,
  allRecipesBadWords,
  tastyRecipesBadWords,
} from './badWords';

describe('badWords', () => {
  it('matches snapshot', () => {
    expect(badWords).toMatchSnapshot();
  });
});

describe('generalBadWords', () => {
  it('matches snapshot', () => {
    expect(generalBadWords).toMatchSnapshot();
  });
});

describe('allRecipesBadWords', () => {
  it('matches snapshot', () => {
    expect(allRecipesBadWords).toMatchSnapshot();
  });
});

describe('tastyRecipesBadWords', () => {
  it('matches snapshot', () => {
    expect(tastyRecipesBadWords).toMatchSnapshot();
  });
});
