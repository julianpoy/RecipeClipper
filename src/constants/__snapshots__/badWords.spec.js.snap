// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allRecipesBadWords matches snapshot 1`] = `
Array [
  "ingredient checklist",
  "instructions checklist",
  "decrease serving",
  "increase serving",
  "adjust",
  "the ingredient list now reflects the servings specified",
  "footnotes",
  "i made it  print",
  "[add all ingredients to shopping list]",
]
`;

exports[`badWords matches snapshot 1`] = `
Array [
  "instructions",
  "directions",
  "procedure",
  "preparation",
  "method",
  "you will need",
  "how to make it",
  "ingredients",
  "total time",
  "active time",
  "prep time",
  "time",
  "yield",
  "servings",
  "notes",
  "select all ingredients",
  "select all",
  "ingredient checklist",
  "instructions checklist",
  "decrease serving",
  "increase serving",
  "adjust",
  "the ingredient list now reflects the servings specified",
  "footnotes",
  "i made it  print",
  "[add all ingredients to shopping list]",
  "scale 1x2x3x",
]
`;

exports[`generalBadWords matches snapshot 1`] = `
Array [
  "instructions",
  "directions",
  "procedure",
  "preparation",
  "method",
  "you will need",
  "how to make it",
  "ingredients",
  "total time",
  "active time",
  "prep time",
  "time",
  "yield",
  "servings",
  "notes",
  "select all ingredients",
  "select all",
]
`;

exports[`tastyRecipesBadWords matches snapshot 1`] = `
Array [
  "scale 1x2x3x",
]
`;
